locales/English/gamecreate.php

```diff
@@ -347,4 +384,45 @@ function setBotFill(){
 		document.getElementById("botBox").checked = false;
 	}
 }
+
+// Display nextPhaseMinutes paragraph only if phaseSwitchPeriod has selected a period.
+nextPhaseMinutesPara = document.getElementById("nextPhaseMinutesPara");
+
+selectPhaseSwitchPeriod = document.getElementById("selectPhaseSwitchPeriod");
+phaseSwitchPeriodPara = document.getElementById("phaseSwitchPeriodPara");
+
+selectPhaseMinutes = document.getElementById("selectPhaseMinutes");
+
+nextPhaseMinutesPara.style.display = "none";
+phaseSwitchPeriodPara.style.display = "none";
+
+
+const updatePhasePeriod = () => {
+	console.log('called!!!');
+	console.log(selectPhaseMinutes.value);

```

Comment:
Let's pull out the console.log statements.

Reply:
Done.